vgscheckout/src/main/java/com/verygoodsecurity/vgscheckout/config/networking/VGSCheckoutPaymentRouteConfig.kt

```diff
@@ -2,24 +2,24 @@ package com.verygoodsecurity.vgscheckout.config.networking
 
 import com.verygoodsecurity.vgscheckout.config.networking.core.CheckoutRouteConfig
 import com.verygoodsecurity.vgscheckout.config.networking.core.VGSCheckoutHostnamePolicy
-import com.verygoodsecurity.vgscheckout.config.networking.request.VGSCheckoutMultiplexingRequestOptions
+import com.verygoodsecurity.vgscheckout.config.networking.request.VGSCheckoutPaymentRequestOptions
 import kotlinx.parcelize.Parcelize
 
 /**
- * Multiplexing checkout route configuration. Only for internal use, as multiplexing request have
- * fixed structure.
+ * Payment orchestration checkout route configuration. Only for internal use,
+ * as Payment orchestration request have fixed structure.
  */
 @Parcelize
-class VGSCheckoutMultiplexingRouteConfig private constructor(
+class VGSCheckoutPaymentRouteConfig private constructor(
```

Comment:
@Endorf This config is used `VGSCheckoutAddCardConfig`, maybe we should add new type of route config? So every type of config will use its own route config?